package.json

```diff
@@ -17,9 +17,10 @@
     "@commitlint/config-conventional": "^17.0.3",
     "@monorepo-utils/workspaces-to-typescript-project-references": "2.8.2",
     "@octokit/core": "4.0.5",
-    "@salesforce/dev-config": "1.6.0",
+    "@salesforce/dev-config": "^3.1.0",
```

Comment:
This required with the typescript version bump.  Only fallout was they have removed the tslint configuration so I had to copy it over to our base configuration file.